tests/components/tesla_wall_connector/test_config_flow.py

```diff
@@ -0,0 +1,207 @@
+"""Test the Tesla Wall Connector config flow."""
+from unittest.mock import patch
+
+from tesla_wall_connector.exceptions import WallConnectorConnectionError
+
+from homeassistant import config_entries, setup
+from homeassistant.components.dhcp import HOSTNAME, IP_ADDRESS, MAC_ADDRESS
+from homeassistant.components.tesla_wall_connector.const import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_SCAN_INTERVAL
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import RESULT_TYPE_CREATE_ENTRY, RESULT_TYPE_FORM
+
+from tests.common import MockConfigEntry
+
+
+async def test_form(mock_wall_connector_version, hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
```

Comment:
We don't need to set up persistent notification integration anymore. We can remove this in all the tests.